docs/language/statements.rst

```diff
@@ -145,3 +142,10 @@ This might be useful when no error string is expected, and will generate shorter
 
 .. include:: ../examples/polkadot/statement_try_catch_no_error_handling.sol
   :code: solidity
+
+.. note::
+
+    At the moment, only catching ``Error`` and ``Panic`` errors with an explicit catch clause are supported.
```

Comment:
Remove "At the moment". 

You're not planing to implement this, are you?

Reply:
Yes and no. My plan is to implement this if either `solc` supports it as well or it's requested by our users.

Reply:
So that is true for every possible Solidity feature: we'll implement if it is requested by users or implemented by solc.

Reply:
- It's making solang look unfinished
- it does not add anything, users don't need to be told "at the moment"
- I think it looks unprofessional

Reply:
FYI I already removed "At the moment" as you suggested :slightly_smiling_face: 

> So that is true for every possible Solidity feature: we'll implement if it is requested by users or implemented by solc.

This would justify leaving "At the moment", because at this moment we don't support, but at the moment it's requested we would support it, wouldn't it? However, it's not important to me.